src/components/HomeScreen.js

```diff
@@ -1,27 +1,95 @@
 import React, { useState } from "react";
-import { StyleSheet, Text, View, Button, Pressable } from "react-native";
-import { StatusBar } from "expo-status-bar";
+import { StyleSheet, Text, View, Button, Image, Pressable } from "react-native";
+import Swiper from 'react-native-swiper';
+import { games } from '../../mock-data/mock-game-data';
+import { users } from '../../mock-data/mock-user-data';
+
+let userGames = games.map(game => {
+  return (
+    <View style={{justifyContent: 'center', alignItems: 'center'}} key={game.title}>
+      <Image
+        source={{uri: game.image}}
+        style={{height: 325, width: 250, marginBottom: 10, borderWidth: 2, borderColor: '#3AE456', borderRadius: 20}}>
+      </Image>
+      <Pressable style={{
```

Comment:
I started with this in the style sheet object, but it was throwing an error. Not sure if it's a scope issue because its trying to run the map before the styles object is declared? not totally sure but this is the only way I could get it to work. open to keep experimenting though